<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Nombre de usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoUsuario">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoPassword">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botonRegistrar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botonRegistrar {
	background-color: rgb(32, 74, 135);
	color: white;
}

#botonRegistrar:hover {
	background-color: rgb(78, 154, 6);
}</string>
    </property>
    <property name="text">
     <string>Registrarme</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 32px;
color: white;</string>
     </property>
     <property name="text">
      <string>REGISTRO</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelIncorrecto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#labelIncorrecto{
	text-align: center;
	font-size: 18px;
	color: red;
}</string>
    </property>
    <property name="text">
     <string>Datos incorrectos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Vuelva a ingresar la contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textoPasswordReiteracion">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botonVolver">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#botonVolver {
	background-color: rgb(92, 53, 102);
	color: white;
}

#botonVolver:hover {
	background-color: rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
